Dockerfile.openshift

```diff
@@ -14,11 +7,26 @@ RUN go build -mod vendor -o bin/whereabouts     cmd/whereabouts.go
 RUN go build -mod vendor -o bin/ip-control-loop cmd/controlloop/controlloop.go
 WORKDIR /
 
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.13
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 AS rhel8
+ADD . /go/src/github.com/k8snetworkplumbingwg/whereabouts
+WORKDIR /go/src/github.com/k8snetworkplumbingwg/whereabouts
+ENV CGO_ENABLED=1
+ENV GO111MODULE=on
+RUN go build -mod vendor -o bin/whereabouts     cmd/whereabouts.go
+RUN go build -mod vendor -o bin/ip-control-loop cmd/controlloop/controlloop.go
+WORKDIR /
+
+FROM registry.ci.openshift.org/ocp/4.13:base
 RUN mkdir -p /usr/src/whereabouts/images && \
-       mkdir -p /usr/src/whereabouts/bin
-COPY --from=builder /go/src/github.com/k8snetworkplumbingwg/whereabouts/bin/whereabouts     /usr/src/whereabouts/bin
-COPY --from=builder /go/src/github.com/k8snetworkplumbingwg/whereabouts/bin/ip-control-loop /usr/src/whereabouts/bin
+       mkdir -p /usr/src/whereabouts/bin && \
+       mkdir -p /usr/src/whereabouts/rhel9/bin && \
+       mkdir -p /usr/src/whereabouts/rhel8/bin
+COPY --from=rhel9 /go/src/github.com/k8snetworkplumbingwg/whereabouts/bin/whereabouts     /usr/src/whereabouts/bin
```

Comment:
Isn't the base image rhel8 here?

Reply:
I suppose this is okey to us because [CNO cni install script](https://github.com/openshift/cluster-network-operator/blob/release-4.13/bindata/network/multus/multus.yaml#L13-L106) will copy appropreate binary from rhel9/rhel8 directory, based on /etc/os-release. `/go/src/github.com/k8snetworkplumbingwg/whereabouts/bin` is only used for fallback.